content/chainguard/libraries/python/build-configuration.md

```diff
@@ -161,7 +161,7 @@ Note the different syntax for `index-url` in the two files.
 
 Refer to the official documentation for [configuring authentication with
 pip](https://pip.pypa.io/en/stable/topics/authentication/) if you are not using
-[.netrc for authentication](/chainguard/libraries/access#netrc).
+[.netrc for authentication](/chainguard/libraries/access/#netrc-for-authentication).
```

Comment:
this is the wrong anchor .. there is an explicit anchor inserted in the page that wont change even if we change title ..

Reply:
thanks for letting me know, fixed these here: https://github.com/chainguard-dev/edu/pull/2447